lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -245,46 +247,61 @@ def test_student_has_access(self):
         course_key = self.course.id
         chapter = BlockFactory.create(category="chapter", parent_location=self.course.location)
         overview = CourseOverview.get_from_id(course_key)
+        subsection = BlockFactory.create(category="sequential", parent_location=chapter.location)
+        unit = BlockFactory.create(category="vertical", parent_location=subsection.location)
+
+        with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred):
+            html_block = BlockFactory.create(
+                category="html",
+                parent_location=unit.location,
+                display_name = "Unpublished Block",
+                data = '<p>This block should not be published.</p>',
+                publish_item = False,
+            )
+            print(self.store.get_branch_setting())
 
+        print(self.store.get_branch_setting())
```

Comment:
Leftover debug code?

Reply:
Definitely, I'll clean this up before squashing.